packages/core-components/src/layout/ErrorPage/ErrorPage.tsx

```diff
@@ -83,16 +106,36 @@ export function ErrorPage(props: IErrorPageProps) {
         <Typography variant="h2" className={classes.title}>
           Looks like someone dropped the mic!
         </Typography>
-        <Typography variant="h6">
+        <Typography variant="h6" className={classes.goBackTitle}>
           <Link to="#" data-testid="go-back-link" onClick={() => navigate(-1)}>
             Go back
           </Link>
           ... or please{' '}
           <Link to={supportUrl || support.url}>contact support</Link> if you
           think this is a bug.
         </Typography>
+        {stack && (
+          <WarningPanel severity="info" subTitle="Stack Trace">
```

Comment:
I'm feeling the warning panel is a bit too visible and distracting tbh, thinking that a small `Show details...` text or something like that would do it.

Reply:
In the sense that we should all together not use the WarningPanel? Or change the title to reflect "Show details..."?

Reply:
By the way, looking at how other errors have been displayed in the Backstage portal I personally feel it makes sense to display this as an error type of warning not info. Thoughts?

<img width="652" alt="Screenshot 2024-01-21 at 7 37 02 PM" src="https://github.com/backstage/backstage/assets/67240543/bdb5092e-4667-4006-a473-6f38b8b357f2">

Reply:
Given that this is very much a user facing page I was thinking something more subtle instead of the warning panel that really sticks out. Something like this:

<img width="1108" alt="image" src="https://github.com/backstage/backstage/assets/4984472/24cf4b48-9c79-4bfd-b717-0cac6f6620e1">